app/assets/javascripts/discourse/app/controllers/sidebar-section-form.js

```diff
@@ -45,16 +45,20 @@ class SectionLink {
     this.router = router;
     this.icon = icon || "link";
     this.name = name;
-    this.value = value ? `${this.protocolAndHost}${value}` : value;
+    this.value = value;
     this.id = id;
   }
 
-  get protocolAndHost() {
-    return window.location.protocol + "//" + window.location.host;
+  get httpHost() {
+    return "http://" + window.location.host;
+  }
+
+  get httpsHost() {
+    return "https://" + window.location.host;
```

Comment:
These don't have to be getters since they do not really depend on any other property. Instead you can just declare them as properties or even as a constant.